File: VB/Helpers/DragDrop/ChartControlHelper.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraCharts
Imports DevExpress.XtraGrid.Columns
Imports System
Imports System.Collections.Generic
Imports System.Linq

Namespace dxQ529392
	Public Class ChartControlHelper
		Inherits BaseDropGridColumnHeaderSupportHelper(Of ChartControl)
		Public Overrides Sub DropObjectCore(ByVal columnFieldName As String, ByVal location As System.Drawing.Point, ByVal control As ChartControl)
			Dim series = New Series() With {.Name = columnFieldName}
			control.Series.Add(series)
		End Sub
	End Class
End Namespace

File: CS/Helpers/DragDrop/ChartControlHelper.cs
using DevExpress.XtraCharts;
using DevExpress.XtraGrid.Columns;
using System;
using System.Collections.Generic;
using System.Linq;

namespace dxQ529392 {
    public class ChartControlHelper : BaseDropGridColumnHeaderSupportHelper<ChartControl> {
        public override void DropObjectCore(string columnFieldName, System.Drawing.Point location, ChartControl control) {
            var series = new Series() { Name = columnFieldName };
            control.Series.Add(series);
        }
    }
}
